<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>874</width>
    <height>655</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>识别工具</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>851</width>
      <height>641</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QCheckBox" name="checkBoxZhiShuGuPiao">
         <property name="minimumSize">
          <size>
           <width>80</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>指数选择</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEditZhiShuGuPiao"/>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonZhiShuGuPiao">
         <property name="text">
          <string>选择</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBoxZhiShuZhangDie">
         <property name="text">
          <string>涨跌幅</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEditZhiShuZhangDieStart">
         <property name="maximumSize">
          <size>
           <width>60</width>
           <height>16777215</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>到</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEditZhiShuZhangDieEnd">
         <property name="maximumSize">
          <size>
           <width>60</width>
           <height>16777215</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QCheckBox" name="checkBoxHangyeGuPiao">
         <property name="minimumSize">
          <size>
           <width>80</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>行业选择</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEditHangYeGuPiao"/>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonHangYeGuPiao">
         <property name="text">
          <string>选择</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBoxHangYeZhangDie">
         <property name="text">
          <string>涨跌幅</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEditHangYeZhangDieStart">
         <property name="maximumSize">
          <size>
           <width>60</width>
           <height>16777215</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>到</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEditHangYeZhangDieEnd">
         <property name="maximumSize">
          <size>
           <width>60</width>
           <height>16777215</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QCheckBox" name="checkBoxGaiNian1GuPiao">
         <property name="minimumSize">
          <size>
           <width>80</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>概念1选择</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEditGaiNian1GuPiao"/>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonGaiNian1GuPiao">
         <property name="text">
          <string>选择</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBoxGaiNian1ZhangDie">
         <property name="text">
          <string>涨跌幅</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEditGaiNian1ZhangDieStart">
         <property name="maximumSize">
          <size>
           <width>60</width>
           <height>16777215</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>到</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEditGaiNian1ZhangDieEnd">
         <property name="maximumSize">
          <size>
           <width>60</width>
           <height>16777215</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QCheckBox" name="checkBoxGaiNian2GuPiao">
         <property name="minimumSize">
          <size>
           <width>80</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>概念2选择</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEditGaiNian2GuPiao"/>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonGaiNian2GuPiao">
         <property name="text">
          <string>选择</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBoxGaiNian2ZhangDie">
         <property name="text">
          <string>涨跌幅</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEditGaiNian2ZhangDieStart">
         <property name="maximumSize">
          <size>
           <width>60</width>
           <height>16777215</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_10">
         <property name="text">
          <string>到</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEditGaiNian2ZhangDieEnd">
         <property name="maximumSize">
          <size>
           <width>60</width>
           <height>16777215</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QCheckBox" name="checkBoxGeGuGuPiao">
         <property name="minimumSize">
          <size>
           <width>80</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>个股选择</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEditGeGuGuPiao"/>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonGeGuGuPiao">
         <property name="text">
          <string>选择</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBoxGeGuZhangDie">
         <property name="text">
          <string>涨跌幅</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEditGeGuZhangDieStart">
         <property name="maximumSize">
          <size>
           <width>60</width>
           <height>16777215</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_11">
         <property name="text">
          <string>到</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEditGeGuZhangDieEnd">
         <property name="maximumSize">
          <size>
           <width>60</width>
           <height>16777215</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <widget class="QRadioButton" name="radioButtonJingqueFind">
         <property name="text">
          <string>精确查找</string>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">buttonGroup</string>
         </attribute>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="radioButtonKeyFind">
         <property name="text">
          <string>含关键字查找</string>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">buttonGroup</string>
         </attribute>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="radioButtonAlgFind">
         <property name="text">
          <string>算法查找</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">buttonGroup</string>
         </attribute>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBoxAllMatch">
         <property name="text">
          <string>全宫相关</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBoxSortDecend">
         <property name="text">
          <string>值降序排</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_6">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_7">
       <item>
        <widget class="QRadioButton" name="radioButtonZhidingWord">
         <property name="text">
          <string>指定字一宫含</string>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">buttonGroup_2</string>
         </attribute>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEditZhiDingOneGongInclude">
         <property name="minimumSize">
          <size>
           <width>130</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>130</width>
           <height>16777215</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>二宫含</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEditZhiDingTwoGongInclude">
         <property name="minimumSize">
          <size>
           <width>130</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>130</width>
           <height>16777215</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_9">
         <item>
          <widget class="QRadioButton" name="radioButtonZhidingDate">
           <property name="text">
            <string>从指定日期获取</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">buttonGroup_2</string>
           </attribute>
          </widget>
         </item>
         <item>
          <widget class="QDateEdit" name="dateEditZhiding"/>
         </item>
         <item>
          <widget class="QRadioButton" name="radioButtonYear">
           <property name="text">
            <string>年</string>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">buttonGroup_3</string>
           </attribute>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="radioButtonMonth">
           <property name="text">
            <string>月</string>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">buttonGroup_3</string>
           </attribute>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="radioButtonDay">
           <property name="text">
            <string>日</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">buttonGroup_3</string>
           </attribute>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_5">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_8">
       <item>
        <widget class="QRadioButton" name="radioButtonAllWord">
         <property name="text">
          <string>全部字一宫含</string>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">buttonGroup_2</string>
         </attribute>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEditAllWordOneGongInclude">
         <property name="minimumSize">
          <size>
           <width>130</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>130</width>
           <height>16777215</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_12">
         <property name="text">
          <string>二宫含</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEditAllWordTwoGongInclude">
         <property name="minimumSize">
          <size>
           <width>130</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>130</width>
           <height>16777215</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_13">
         <property name="text">
          <string>一宫不含</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEditAllWordOneGongExclude">
         <property name="minimumSize">
          <size>
           <width>130</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>130</width>
           <height>16777215</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_14">
         <property name="text">
          <string>二宫不含</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEditAllWordTwoGongExclude">
         <property name="minimumSize">
          <size>
           <width>130</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>130</width>
           <height>16777215</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_4">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_10">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_11">
         <item>
          <widget class="QLabel" name="label_20">
           <property name="text">
            <string>识别内容从</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDateEdit" name="dateEditRecgStart"/>
         </item>
         <item>
          <widget class="QLabel" name="label_21">
           <property name="text">
            <string>到</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDateEdit" name="dateEditRecgEnd"/>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QLabel" name="label_15">
         <property name="text">
          <string>   值算法：个股值 - 概念2</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEditGaiNian2Percent">
         <property name="maximumSize">
          <size>
           <width>40</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>0</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_16">
         <property name="text">
          <string>% - 概念1</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEditGaiNian1Percent">
         <property name="maximumSize">
          <size>
           <width>40</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>0</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_17">
         <property name="text">
          <string>% - 行业</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEditHangyePercent">
         <property name="maximumSize">
          <size>
           <width>40</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>0</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_18">
         <property name="text">
          <string>% - 指数</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEditZhiShuPercent">
         <property name="maximumSize">
          <size>
           <width>40</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>0</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_19">
         <property name="text">
          <string>%</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_12">
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonStart">
         <property name="text">
          <string>开始识别</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonSave">
         <property name="text">
          <string>存储</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QTextEdit" name="textEditLog">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
  <buttongroup name="buttonGroup_2"/>
  <buttongroup name="buttonGroup_3"/>
 </buttongroups>
</ui>
